docs/topics/tutorials-and-resources.md

```diff
@@ -0,0 +1,106 @@
+* [ViewSets and Routers - Django REST Framework Part 3][viewsets-and-routers-django-rest-framework-part-3]
+* [Django REST Framework User Endpoint][django-rest-framework-user-endpoint]
+* [Check Credentials Using Django REST Framework][check-credentials-using-django-rest-framework]
+* [Creating a Production Ready API with Python and Django REST Framework – Part 1][creating-a-production-ready-api-with-python-and-drf-part1]
+* [Creating a Production Ready API with Python and Django REST Framework – Part 2][creating-a-production-ready-api-with-python-and-drf-part2]
+
+
+## Videos
+
+### Talks
+
+* [How to Make a Full Fledged REST API with Django OAuth Toolkit][full-fledged-rest-api-with-django-oauth-tookit]
+* [Django REST API - So Easy You Can Learn It in 25 Minutes][django-rest-api-so-easy]
+* [Tom Christie about Django Rest Framework at Django: Under The Hood][django-under-hood-2014]
+* [Django REST Framework: Schemas, Hypermedia & Client Libraries][pycon-uk-2016]
+
+
+### Tutorials
+
+
+* [Django REST Framework Part 1][django-rest-framework-part-1-video]
+* [Django REST Framework in Your PJ's!][drf-in-your-pjs]
+* [Building a REST API Using Django & Django REST Framework][building-a-rest-api-using-django-and-drf]
+* [Blog API with Django REST Framework][blog-api-with-drf]
+* [Ember and Django Part 1][ember-and-django-part 1-video]
+* [Django REST Framework Image Upload Tutorial (with AngularJS)][drf-image-upload-tutorial-with-angularjs]
+* [Django REST Framework Tutorials][drf-tutorials]
+
+
+## Articles
+
+* [Web API performance: Profiling Django REST Framework][web-api-performance-profiling-django-rest-framework]
+* [API Development with Django and Django REST Framework][api-development-with-django-and-django-rest-framework]
+* [Integrating Pandas, Django REST Framework and Bokeh][integrating-pandas-drf-and-bokeh]
+* [Controlling Uncertainty on Web Applications and APIs][controlling-uncertainty-on-web-apps-and-apis]
+* [Full Text Search in Django REST Framework with Database Backends][full-text-search-in-drf]
+* [OAuth2 Authentication with Django REST Framework and Custom Third-Party OAuth2 Backends][oauth2-authentication-with-drf]
+* [Nested Resources with Django REST Framework][nested-resources-with-drf]
+* [Image Fields with Django REST Framework][image-fields-with-drf]
+* [Chatbot Using Django REST Framework + api.ai + Slack — Part 1/3][chatbot-using-drf-part1]
+* [New Django Admin with DRF and EmberJS... What are the News?][new-django-admin-with-drf-and-emberjs]
+* [Blog posts about Django REST Framework][medium-django-rest-framework]
+
+## Books
+
+* [Hello Web App: Intermediate Concepts, Chapter 10][hello-web-app-intermediate]
+
+### Documentations
+* [Classy Django REST Framework][cdrf.co]
+* [DRF-schema-adapter][drf-schema]
+
+Want your Django REST Framework talk/tutorial/article to be added to our website? Please [email us][mailto:anna@django-rest-framework.org]!
```

Comment:
Also thinking about it something like "or know of a resource that's not yet included here?" would be good too.